### ISSUE TITLE
Proposal for Translating Fastify Website into Spanish

### ISSUE THREAD

**Issue by moradebianchetti81** [fastify/website#219]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the issue has not already been raised

### Issue

hi, i noticed that the fastify website is currently only in English, i wanted to see if there was an idea to translate it into another language such as Spanish.

**Comment by Uzlopak** [comment]
What is the goal of having the readme in spanish, if we can not ensure that we keep it up to date, as not every contributor can write spanish. Also spanish has more writing styles... Should we use argentinian spanish? gallego spanish? ecuadorian spanish? philipinian spanish?

**Comment by Eomm** [comment]
TBH, if we can leverage AI to translate the website at build time it would be cool.

Doing it manually leads to unmaintaned docs. eg:

- Portuguese 5y without updates https://github.com/fastify/docs-portuguese
- Chinese 4y without updates https://github.com/fastify/docs-chinese

**Comment by gurgunday** [comment]
I think we should archive those 😅

I like the idea of ai generated translations at build time

**Comment by mcollina** [comment]
I think we _could_ maintain a multilingual website, however we lack the resources to maintain multilingual docs.

I agree on archiving those repos.

**Comment by metcoder95** [comment]
We should be careful of these automated translations, especially over complex languages with different dialects like Spanish (I've seen such gibberish from AI translations).

I'd say that if we can find a willing volunteer who can maintain those translations (not necessarily write all of them, but keep it up to date and flowing) it should be nice to have multilingual support.

Not sure if @moradebianchetti81 would like to volunteer, but I can definitely help with the Spanish one if we want to give it a try with a pilot language

**Comment by Fdawgs** [comment]
Don't most browsers have translation features built in?

For example, visiting https://www.nintendo.com/jp causes the following the pop up in Edge:
![image](https://github.com/fastify/website/assets/43814140/104c52ff-5416-40bb-9ab7-4a5eb39dfd51)

**Comment by gurgunday** [comment]
> Don't most browsers have translation features built in?

Yeah I actually wanted to build a product around this and Chrome/Edge auto translate killed my motivation 😂

**Comment by jean-michelet** [comment]
Rather than software maintainers making the effort to translate, people can make the effort to learn how to read English.
Might be useful for a developer ^^

And yes, they are tons of add-ons to translate, with deepl the translation is very good.
